loader/CMakeLists.txt

```diff
@@ -315,6 +320,33 @@ else()
         add_library(vulkan STATIC ${NORMAL_LOADER_SRCS} ${OPT_LOADER_SRCS})
     else()
         add_library(vulkan SHARED ${NORMAL_LOADER_SRCS} ${OPT_LOADER_SRCS})
+        if(BUILD_WSI_XCB_SUPPORT)
```

Comment:
FYI this code branch will break on apple because the previous if is for apple *and* static loader. I think the added code just needs to have the `end()` for the else() on line 321 moved to line 324.